flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecPythonGroupWindowAggregate.java

```diff
@@ -343,4 +444,60 @@ public StreamExecPythonGroupWindowAggregate(
                     e);
         }
     }
+
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    private OneInputStreamOperator<RowData, RowData>
+            getGeneralPythonStreamGroupWindowAggregateFunctionOperator(
```

Comment:
It seems we still need to support `allowLateness` in the GeneralPythonStreamGroupWindowAggregateFunctionOperator in order to align with java window.

Reply:
Yes. We still need `allowLateness `